subiquity/common/types.py

```diff
@@ -341,10 +341,14 @@ class StorageResponse:
 
 @attr.s(auto_attribs=True)
 class StorageResponseV2:
-    disks: List[Disk]
-    need_root: bool  # if True, there is not yet a partition mounted at "/"
-    need_boot: bool  # if True, there is not yet a boot partition
-    install_minimum_size: int
+    status: ProbeStatus
+    error_report: Optional[ErrorReportRef] = None
```

Comment:
`error_report` is defined twice in this class.
It's sad that we can't reliably use mypy with subiquity yet because it would catch this kind of thing.

Reply:
Well that was silly, thanks!